tatogenerator.py

```diff
@@ -2,16 +2,25 @@
 
 import time
 import random
+import argparse
+
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
```

Comment:
ArgumentParser() - how is it working, I couldn't find definition of this method in this script